pysat/tests/test_instrument.py

```diff
@@ -564,7 +712,81 @@ def teardown(self):
         """Runs after every method to clean up previous testing."""
         del self.testInst
 
+#------------------------------------------------------------------------------
+#
+# Repeat TestBasics above with shifted file dates
+#
+#------------------------------------------------------------------------------
+
+class TestBasicsShiftedFileDates(TestBasics):
+    def setup(self):
+        reload(pysat.instruments.pysat_testing)
```

Comment:
use `re_load` here (see Lines 12-15 for version-dependent import statement)

Reply:
Fixed.